lib/models/rocket.py

```diff
@@ -1,16 +1,18 @@
+from enum import Enum
 from typing import Optional, Tuple, List
 from pydantic import BaseModel, PrivateAttr
-from enum import Enum
 from lib.models.motor import Motor
 from lib.models.aerosurfaces import Fins, NoseCone, Tail, RailButtons
 from lib.models.parachute import Parachute
 
+
 class RocketOptions(str, Enum):
     calisto: str = "Calisto"
     custom: str = "Custom"
```

Comment:
I never saw Enums' members being defined using lower-case, I did some searches and it seems that Python only "highly recommend" using UPPER_CASE for these members.

https://docs.python.org/3/howto/enum.html#enum-cookbook

![image](https://github.com/RocketPy-Team/Infinity-API/assets/63590233/64906685-8310-4302-8fec-65d3fb957a8c)

Reply:
Didn't know that.

Despite having no chance of cluttering in this specific case, I will follow the recommendation for the sake of transparency.